### ISSUE TITLE
Stuck at loading samples

### ISSUE THREAD

**Issue by dausdanger** [iperov/DeepFaceLab#5459]
![Screenshot 2022-01-10 082744](https://user-images.githubusercontent.com/74890153/148712369-32217157-4355-4913-9c7e-376c6c5c981d.png)
Im using RTX 3070 with latest dfl

**Comment by SrGrafoasspee** [comment]
Same problem

Running trainer.

[new] No saved models found. Enter a name of a new model : sneed
sneed

Model first run.

Choose one or several GPU idxs (separated by comma).

[CPU] : CPU
  [0] : NVIDIA GeForce RTX 3060 Laptop GPU

[0] Which GPU indexes to choose? :
0

[1] Autobackup every N hour ( 0..24 ?:help ) : 1
1
[n] Write preview history ( y/n ?:help ) :
n
[100000] Target iteration :
100000
[n] Flip SRC faces randomly ( y/n ?:help ) :
n
[n] Flip DST faces randomly ( y/n ?:help ) :
n
[2] Batch_size ( ?:help ) :
2
[128] Resolution ( 64-640 ?:help ) :
128
[f] Face type ( h/mf/f/wf/head ?:help ) :
f
[df] AE architecture ( ?:help ) : ?

'df' keeps more identity-preserved face.
'liae' can fix overly different face shapes.
'-u' increased likeness of the face.
'-d' (experimental) doubling the resolution using the same computation cost.
Examples: df, liae, df-d, df-ud, liae-ud, ...

[df] AE architecture ( ?:help ) : liae-u
liae-u
[256] AutoEncoder dimensions ( 32-1024 ?:help ) :
256
[64] Encoder dimensions ( 16-256 ?:help ) :
64
[64] Decoder dimensions ( 16-256 ?:help ) :
64
[22] Decoder mask dimensions ( 16-256 ?:help ) :
22
[n] Eyes and mouth priority ( y/n ?:help ) :
n
[n] Uniform yaw distribution of samples ( y/n ?:help ) :
n
[n] Blur out mask ( y/n ?:help ) :
n
[y] Place models and optimizer on GPU ( y/n ?:help ) :
y
[y] Use AdaBelief optimizer? ( y/n ?:help ) :
y
[n] Use learning rate dropout ( n/y/cpu ?:help ) :
n
[y] Enable random warp of samples ( y/n ?:help ) :
y
[0.0] Random hue/saturation/light intensity ( 0.0 .. 0.3 ?:help ) :
0.0
[0.0] GAN power ( 0.0 .. 5.0 ?:help ) :
0.0
[0.0] Face style power ( 0.0..100.0 ?:help ) :
0.0
[0.0] Background style power ( 0.0..100.0 ?:help ) :
0.0
[none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) :
none
[n] Enable gradient clipping ( y/n ?:help ) :
n
[n] Enable pretraining mode ( y/n ?:help ) :
n
Initializing models: 100%|###############################################################| 5/5 [00:01<00:00,  2.84it/s]
Loaded 25461 packed faces from D:\***\workspace\data_src\aligned
Loaded 25603 packed faces from D:\***\workspace\data_dst\aligned

**Comment by luckyycode** [comment]
Increase virtual memory size, helped me
I set it to 28 gb

**Comment by Laevon** [comment]
I changed my nvidia driver to studio version, and it began to work

**Comment by ghost** [comment]
Did you ever find the answer? If so, would you mind sharing it and closing this issue?

**Comment by yangdeai** [comment]
> Same problem
> 
> Running trainer.
> 
> [new] No saved models found. Enter a name of a new model : sneed sneed
> 
> Model first run.
> 
> Choose one or several GPU idxs (separated by comma).
> 
> [CPU] : CPU [0] : NVIDIA GeForce RTX 3060 Laptop GPU
> 
> [0] Which GPU indexes to choose? : 0
> 
> [1] Autobackup every N hour ( 0..24 ?:help ) : 1 1 [n] Write preview history ( y/n ?:help ) : n [100000] Target iteration : 100000 [n] Flip SRC faces randomly ( y/n ?:help ) : n [n] Flip DST faces randomly ( y/n ?:help ) : n [2] Batch_size ( ?:help ) : 2 [128] Resolution ( 64-640 ?:help ) : 128 [f] Face type ( h/mf/f/wf/head ?:help ) : f [df] AE architecture ( ?:help ) : ?
> 
> 'df' keeps more identity-preserved face. 'liae' can fix overly different face shapes. '-u' increased likeness of the face. '-d' (experimental) doubling the resolution using the same computation cost. Examples: df, liae, df-d, df-ud, liae-ud, ...
> 
> [df] AE architecture ( ?:help ) : liae-u liae-u [256] AutoEncoder dimensions ( 32-1024 ?:help ) : 256 [64] Encoder dimensions ( 16-256 ?:help ) : 64 [64] Decoder dimensions ( 16-256 ?:help ) : 64 [22] Decoder mask dimensions ( 16-256 ?:help ) : 22 [n] Eyes and mouth priority ( y/n ?:help ) : n [n] Uniform yaw distribution of samples ( y/n ?:help ) : n [n] Blur out mask ( y/n ?:help ) : n [y] Place models and optimizer on GPU ( y/n ?:help ) : y [y] Use AdaBelief optimizer? ( y/n ?:help ) : y [n] Use learning rate dropout ( n/y/cpu ?:help ) : n [y] Enable random warp of samples ( y/n ?:help ) : y [0.0] Random hue/saturation/light intensity ( 0.0 .. 0.3 ?:help ) : 0.0 [0.0] GAN power ( 0.0 .. 5.0 ?:help ) : 0.0 [0.0] Face style power ( 0.0..100.0 ?:help ) : 0.0 [0.0] Background style power ( 0.0..100.0 ?:help ) : 0.0 [none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) : none [n] Enable gradient clipping ( y/n ?:help ) : n [n] Enable pretraining mode ( y/n ?:help ) : n Initializing models: 100%|###############################################################| 5/5 [00:01<00:00, 2.84it/s] Loaded 25461 packed faces from D:_\workspace\data_src\aligned Loaded 25603 packed faces from D:_\workspace\data_dst\aligned



I met the same problem, and solved it by change the VRAM to 128Gb.
